apps/web/pages/_app.page.tsx

```diff
@@ -58,6 +58,7 @@ export default withUrqlClient(
         cookie: context ? context.req?.headers.cookie ?? '' : document.cookie,
       },
     }),
+    requestPolicy: 'cache-and-network',
```

Comment:
can you double check the other bug if this is solved then....

Reply:
Remove the context useMemo everywhere because it is not needed any more now.

Reply:
useMemo still needed in case of staying on the same page